src/components/latency_test.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react"
+
+const states = {
+  INIT: "init",
+  TESTING: "testing",
+  TESTED: "tested",
+  ERROR: "error",
+}
+
+const getLatencyToCloudflare = async () => {
+  const TEST_URL = "https://1.1.1.1/cdn-cgi/trace"
+  const NUM_TESTS = 5
+
+  let started = 0
+  async function load() {
+    started++
+
+    await fetch(TEST_URL, {
+      mode: "no-cors",
+      cache: "no-store",
+    })
```

Comment:
i _think_ this fetch call might be made 6 times instead of 5 may want to change the `<= NUM_TESTS` to `< NUM_TESTS` (maybe console.log it to double check how many times it's running i just ran through it in my head)